packages/bot-web-ui/src/app/app.jsx

```diff
@@ -12,20 +12,34 @@ import {
 } from 'Components';
 import BlocklyLoading from '../components/blockly-loading';
 import { MobxContentProvider } from 'Stores/connect';
-import RootStore from 'Stores';
+import { StoreProvider, useStore } from '@deriv/stores';
+import { DBotStoreProvider, useDBotStore } from 'Stores/dbotStore';
 import GTM from 'Utils/gtm';
 import BotBuilder from 'Components/dashboard/bot-builder';
 import './app.scss';
 
-const App = ({ passthrough }) => {
+const AppWrapper = ({ passthrough }) => {
```

Comment:
@shafin-deriv I suggest creating a `DBotProviders` component same as what we have in `packages/cashier/src/cashier-providers.tsx` and move everything there 🙇🏻